.circleci/config.yml

```diff
@@ -7,26 +7,24 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:8.6.0
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/mongo:3.4.4
+      - image: circleci/node:10
 
     working_directory: ~/repo
 
     steps:
       - checkout
 
+      - run: curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.7.0
+
       # Download and cache dependencies
       - restore_cache:
           keys:
           - v1-dependencies-{{ checksum "package.json" }}
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: yarn install
+      - run: yarn
+      - run: npm rebuild node-sass --force
```

Comment:
Why do we need to rebuild `node-sass`? Maybe add a comment.

Reply:
Trying to remove it, otherwise Ill add a comment